sma-em/run.py

```diff
@@ -34,24 +44,22 @@ def main():
     LOOP = asyncio.get_event_loop()
     LOOP.set_debug(True)
 
-    sock = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
-    sock.bind(("", BROADCAST_PORT))
-    group = socket.inet_aton(BROADCAST_ADDR)
-    mreq = struct.pack("4sL", group, socket.INADDR_ANY)
-    sock.setsockopt(socket.IPPROTO_IP, socket.IP_ADD_MEMBERSHIP, mreq)
-    # sock.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
-    # sock.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEPORT, 1)
+    RECONNECT_INTERVAL = sensors.OPTIONS["RECONNECT_INTERVAL"]
```

Comment:
Lets use the const & default

```suggestion
    RECONNECT_INTERVAL = sensors.OPTIONS.get(sensors.RECONNECT_INTERVAL, 86400)
```